chunky/src/java/se/llbit/chunky/block/MinecraftBlockProvider.java

```diff
@@ -994,7 +994,14 @@ public Collection<String> getSupportedBlocks() {
 
   @Override
   public Block getBlockByTag(String namespacedName, Tag tag) {
-    String name = namespacedName.substring(10); // drop the minecraft: prefix
+    // drop the minecraft: prefix
+    String[] split = namespacedName.split(":", 2); // split into maximum 2 parts
+
+    //TODO: modded block support? non-minecraft prefix is ignored
+    if(split.length != 2 || !split[0].equals("minecraft")) {
+      return null;
+    }
```

Comment:
Actually, this already _is_ modded block support. Chunky will try all registered `BlockProvider`s and use the first one that returns a non-null value.

```suggestion
    if(split.length != 2 || !split[0].equals("minecraft")) {
      return null;
    }
```